pom.xml

```diff
@@ -78,6 +81,25 @@
             <version>${liquibase.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
```

Comment:
spring-boot-starter-validation provides this dependency, so if yout have one, don't need to add it